pages/scenario_selection_page.py

```diff
@@ -309,67 +381,344 @@ def scenario_selection_page():
+        lifetime_costs["life_span"]
+        .mean()
+        .round()
+        .astype(int)
+    )
+    lifetime_costs = pd.concat(
+        [lifetime_costs, weighted_lifetime_costs]
+    ).reset_index(drop=True)
+
+
+    lifetime_costs_selected_archetypes = lifetime_costs[
+        lifetime_costs["archetype_label"].isin(filter_archetypes)
     ]
 
-    st.dataframe(dummy_data, use_container_width=True)
-    st.dataframe(total_dummy_data, use_container_width=True)
+    df_comparing_costs = lifetime_costs_selected_archetypes.melt(
+        id_vars=["archetype_label", "technology", "life_span"],
+        value_vars=[
+            "lifetime_running_costs",
+            "installation_costs_after_subsidy",
+            "loan_interest",
+            "lifetime_maintenance_costs",
+        ],
+        var_name="cost_type",
+        value_name="cost",
+    )
+
+    cost_type_name_mapping = {
+        "lifetime_maintenance_costs": "Maintenance costs",
+        "loan_interest": "Loan interest",
+        "installation_costs_after_subsidy": "Installation costs (after subsidy)",
+        "lifetime_running_costs": "Running costs",
+
+    }
 
-    csv_annual = dummy_data.to_csv()
-    csv_total = total_dummy_data.to_csv()
+    df_comparing_costs["cost_type"] = df_comparing_costs["cost_type"].map(cost_type_name_mapping)
+
+    # Convert column to ordered categorical
+    df_comparing_costs["archetype_label"] = pd.Categorical(
+        df_comparing_costs["archetype_label"],
+        categories=mapped_archetype_options + ["Weighted average archetype"],
+        ordered=True
+    )
+    df_comparing_costs["cost_type"] = pd.Categorical(
+        df_comparing_costs["cost_type"],
+        categories=list(cost_type_name_mapping.values()),
+        ordered=True
+    )
+    df_comparing_costs = df_comparing_costs.sort_values(by=["archetype_label", "cost"])
+
+    cost_component_order = {
+        "Loan interest":0,
+        "Maintenance costs":1,
+        "Installation costs (after subsidy)":2,
+        "Running costs":3,
+    }
+    df_comparing_costs["cost_type_order"] = df_comparing_costs["cost_type"].map(cost_component_order)
+
+    chart = (
+        alt.Chart(df_comparing_costs)
+        .mark_bar()
+        .encode(
+            y=alt.Y("technology:N", axis=alt.Axis(title=None)),
+            x=alt.X("sum(cost):Q", axis=alt.Axis(title="£")),
+            color=alt.Color(
+                "cost_type:N",
+                title="Cost component",
+                scale=alt.Scale(
+                    domain=cost_type_name_mapping.values(),
+                    range=[
+                        NESTA_COLOURS[10],
+                        NESTA_COLOURS[2],
+                        NESTA_COLOURS[1],
+                        NESTA_COLOURS[0],
+                    ],
+                ),
+            ),
+            order=alt.Order(
+                "cost_type_order:N",
+                sort="descending"
+            )
+        )
```

Comment:
Adding the new `total_cost` value to be displayed in tooltips

```suggestion
            order=alt.Order(
                "cost_type_order:N",
                sort="descending"
            ),
            tooltip=[
                alt.Tooltip("archetype_label:N", title="Archetype"),
                alt.Tooltip("technology:N", title="Technology"),
                alt.Tooltip("cost_type:N", title="Cost component type"),
                alt.Tooltip("cost:Q", title="Cost component (£)", format=",.2f"),
                alt.Tooltip("total_cost:Q", title="Total cost (£)", format=",.2f"),
            ],
        )
```